src/patternfly/components/Nav/nav.scss

```diff
@@ -130,6 +128,10 @@
   &.pf-m-fill {
     flex-grow: 1;
   }
+
+  + .#{$menu} {
```

Comment:
This is due to how the markup is setup in Core - a nav with a menu sibling. In React, though, the menu is a descendant of nav.

If the React markup is the correct one, I can update the nav flyout example in core to match and update this to remove this sibling combinator and just target any menu decendant of nav.

Reply:
Updated to nsst the menu inside nav per off-GH discussion